examples/quick-start/packages/server/src/index.ts

```diff
@@ -83,7 +83,7 @@ const findContent = makeDefaultFindContent({
 const makeUserMessage: MakeUserMessageFunc = async function ({
   content,
   originalUserMessage,
-}): Promise<OpenAiChatMessage & { role: "user" }> {
+}): Promise<UserMessage> {
```

Comment:
```suggestion
}) {
```

we can actually simplify this further by just removing the return type since it's inherited from the `MakeUserMessageFunc`